Main CI red: Deploy to staging

## Main CI red

**Repo**: alexsiri7/reli
**SHA**: `2b78e9da13596997f8b91ce29028f571f0ed7cb5`
**Run**: https://github.com/alexsiri7/reli/actions/runs/24987713073
**Failed jobs**: Deploy to staging

Auto-filed by `pipeline-health-cron.sh`. Main CI is a pipeline bottleneck, so archon has been fired immediately on this issue rather than queued. This issue is tagged `archon:in-progress` so the regular pickup cron will not double-fire.

### Steps
1. `gh run view 24987713073 --repo alexsiri7/reli --log-failed`
2. Identify root cause (dep bump? toolchain? flaky test?)
3. Fix on a branch, open PR, ensure CI goes green

## Manual action required — relabeling `archon:blocked`

`pipeline-health-cron` fired `archon-assist` after detecting a 30-min lull in main-branch progress. Diagnosis below.

### What's actually wrong

The latest **Staging → Production Pipeline** run on `main` ([run 25007185051](https://github.com/alexsiri7/reli/actions/runs/25007185051), commit `3dfb995c`) failed at the *Validate Railway secrets* step with:

```
##[error]RAILWAY_TOKEN is invalid or expired: Not Authorized
Rotate the token — see DEPLOYMENT_SECRETS.md Token Rotation section.
```

The Railway secrets *are* set in GitHub Actions (the validation script saw all four), but the token itself is rejected by Railway's API. PR #738 (which I'd previously shipped to add the auth check) is doing exactly what it was designed to do — surfacing the rotted token instead of letting the deploy fail later with a confusing error.

### Why archon kept spinning on this

Each archon run on #733 rediscovers that the fix is "rotate the token in GitHub Actions secrets," exits, then ~2 hours later `issue-pickup-cron` sees a stale `archon:in-progress` label and re-queues. That's a no-op loop — archon can't write GitHub Actions secrets.

### Action needed (you, manually)

1. Generate a new Railway API token: https://railway.com/account/tokens
2. Update the `RAILWAY_TOKEN` secret in https://github.com/alexsiri7/reli/settings/secrets/actions
3. Re-run the failed workflow (or push any commit) to verify the staging deploy goes green
4. Once green, change this issue's label back from `archon:blocked` to `archon:queued` (or just close it)

### What I changed

- Relabeled this issue `archon:queued` → `archon:blocked` to break the re-pickup cycle. The pickup cron treats `archon:blocked` as managed-but-don't-touch.
- No code changes.

### Other repos checked — all healthy
- un-reminder, cosmic-match, word-coach-annie, filmduel, interstellarai.net: green or already-tracked deploy issues; no stuck work.
- The 18:00 "no progress" alarm was a quiet 30-min window after PR #738 merged at 16:27 — not a real outage.

--- a/backend/agents.py
+++ b/backend/agents.py
@@ -754,6 +754,11 @@ the user's message tone — short and direct gets consultant energy, exploratory
 and reflective gets coaching warmth.
 """
 
+_STYLE_OVERLAYS: dict[str, str] = {
+    "coach": _RESPONSE_COACH_OVERLAY,
+    "consultant": _RESPONSE_CONSULTANT_OVERLAY,
+}
+
 
 def load_personality_preferences(user_id: str) -> list[dict[str, Any]]:
     """Load personality preference patterns from Things with type_hint='preference'.
@@ -826,12 +831,7 @@ def get_response_system_prompt(
     personality_patterns: list[dict[str, Any]] | None = None,
 ) -> str:
     """Return the response agent system prompt with the appropriate style overlay."""
-    if interaction_style == "coach":
-        prompt = RESPONSE_AGENT_SYSTEM + _RESPONSE_COACH_OVERLAY
-    elif interaction_style == "consultant":
-        prompt = RESPONSE_AGENT_SYSTEM + _RESPONSE_CONSULTANT_OVERLAY
-    else:
-        prompt = RESPONSE_AGENT_SYSTEM + _RESPONSE_AUTO_OVERLAY
+    prompt = RESPONSE_AGENT_SYSTEM + _STYLE_OVERLAYS.get(interaction_style, _RESPONSE_AUTO_OVERLAY)
 
     if personality_patterns:
         prompt += _build_personality_overlay(personality_patterns)

--- a/backend/routers/chat.py
+++ b/backend/routers/chat.py
@@ -235,14 +235,14 @@ def list_sessions(user_id: str = Depends(require_user)) -> list[ChatSessionSumma
         rows = session.exec(stmt).all()
     return [
         ChatSessionSummary(
-            id=row[0].id,
-            title=row[0].title,
-            origin=row[0].origin,
-            created_at=row[0].created_at,
-            last_active_at=row[0].last_active_at,
-            message_count=row[1],
+            id=record.id,
+            title=record.title,
+            origin=record.origin,
+            created_at=record.created_at,
+            last_active_at=record.last_active_at,
+            message_count=count,
         )
-        for row in rows
+        for record, count in rows
     ]
 
 

--- a/frontend/src/store.ts
+++ b/frontend/src/store.ts
@@ -464,7 +464,9 @@ function _preferenceConfidenceLabel(data: unknown): string {
   const d = data as Record<string, unknown>
   if (typeof d.confidence === 'number') {
     const c = d.confidence as number
-    return c >= 0.7 ? 'strong' : c >= 0.5 ? 'moderate' : 'emerging'
+    if (c >= 0.7) return 'strong'
+    if (c >= 0.5) return 'moderate'
+    return 'emerging'
   }
   if (Array.isArray(d.patterns) && d.patterns.length > 0) {
     const first = d.patterns[0] as Record<string, unknown>

